README.md

```diff
@@ -33,7 +35,6 @@ This repository contains all submitted Centriuge Proposals. The genesis proposal
 ## Proposed CPs
 |CP | Short title |
 |---|-------------|
-|[CP51](./cps/CP51.md)  | Onboarding the 2nd batch of collators  |
-|[CP52](./cps/CP52.md)  | Changes to our CP Framework and Governance Process  |
+
 |[CP53](./cps/CP53.md)  | Listing CFG in HydraDX Omnipool & seeding initial liquidity  |
```

Comment:
I think the table is broken . Check the spaces.

Reply:
@ImdioR it's fixed now - there was an extra line break so it didn't render properly.